docs/components/movement-sensor/_index.md

```diff
@@ -683,7 +630,7 @@ For more information, see the [Go SDK Code](https://github.com/viamrobotics/rdk/
 
 ## Next Steps
 
-Try adding a movement sensor to your [mobile robot](../base/) and writing some code with our [SDKs](../../program/sdk-as-client/) to implement closed-loop movement control for your robot.
+Try adding a movement sensor to your [mobile robot](/components/base) and writing some code with our [SDKs](/program/sdk-as-client) to implement closed-loop movement control for your robot.

```

Comment:
Why this change? I thought we are doing relative links now so that they work in the PR builds?

Reply:
Good catch changed :)